app/services/application_service.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationService
+  def self.perform(*args, &block)
+    new(*args, &block).perform
+  end
+end
```

Comment:
This class helps solidify an established pattern we already have in our code...
all of our services have a single entry point (which makes it clear how to use them)
To implement that they each have an object method called `perform` (self.perform) which instantiates the object with all the passed in parameters and then calls the `perform` method on  object itself (which is where all the logic is created).